apps/web/src/lib/auth/index.ts

```diff
@@ -30,10 +30,10 @@ export const { handlers, signIn, signOut, auth } = NextAuth({
           .update(user)
           .set({
             firstName: params.profile.given_name,
-            lastName: params.profile.family_name,
+            lastName: params.profile.family_name || "",
             photoUrl: params.profile.picture,
             // keep the name in sync
-            name: `${params.profile.given_name} ${params.profile.family_name}`,
+            name: ${params.profile.given_name}${params.profile.family_name ? " " + params.profile.family_name : ""},
```

Comment:
Can we use template literal here please :)

Reply:
```suggestion
            name: `${params.profile.given_name} ${params.profile.family_name || " "}`.trim(),
```